deployments/ansible/roles/splunk-otel-collector/README.md

```diff
@@ -0,0 +1,119 @@
+# Role Name
+
+Ansible role that installs Splunk OpenTelemetry Connector configured to
+collect metrics, traces and logs from Linux machines and send data to [Splunk 
+Observability Cloud](https://www.splunk.com/en_us/observability.html). 
+
+## Prerequisites
+
+- [Splunk Access Token](https://docs.splunk.com/Observability/admin/authentication-tokens/org-tokens.html#admin-org-tokens)
+- [Splunk Realm](https://dev.splunk.com/observability/docs/realms_in_endpoints/)
+- [Double-check exposed ports](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/security.md#exposed-endpoints) 
+  to make sure your environment doesn't have conflicts. Ports can be changed in the collector's configuration.
+
+## Usage
+
+To install the role, clone the agent repo to your control host and add the 
+`splunk-otel-collector` role directory to its `roles_path` in `ansible.cfg`, 
+or use this document's directory as your working directory.
+
+To use this role, simply include the `splunk-otel-collector` role invocation 
+in your playbook. The following example shows how to use the role in a
+playbook with minimal required configuration:
+
+```yaml
+- name: Install Splunk OpenTelemetry Connector
+  hosts: all
+  become: yes
+  vars:
+    splunk_access_token: YOUR_ACCESS_TOKEN
+    splunk_realm: SPLUNK_REALM
+  tasks:
+    - name: "Include splunk-otel-collector"
+      include_role:
+        name: "splunk-otel-collector"
+```
+
+## Role Variables
+
+- `splunk_access_token` (**Required**): The Splunk access token to
+  authenticate requests.
+
+- `splunk_realm`: Which realm to send the data to. The `SPLUNK_REALM`
+  environment variable will be set with this value for the Splunk OTel 
+  Collector service. (**default:** `us0`)
```

Comment:
I think we should get rid of a default realm and make it required (same in the helm chart) since if a user is in a different realm it deploys but nothing works and they're confused why. I am one of those users. :)

Reply:
Let's do it as a separate effort along  with other deployments?

Reply:
When the default is removed, detailing examples (or all supported) could be helpful